js/config.js

```diff
@@ -41,6 +41,7 @@ function getUserConfigField(fieldName) {
             return config[fieldName];
         }
 
+        log.send(logLevels.ERROR, 'config: field does not exist in user config: ' + fieldName);
```

Comment:
i don't think we want an error log here, in certain cases it legitimately will only be in global config, so this might be hit.  error handling for this cases should be at a higher level where it doesn't appear in either.